test-app/package.json

```diff
@@ -22,12 +16,22 @@
     "test:ember": "ember test",
     "test:all": "ember try:each"
   },
-  "devDependencies": {
-    "@ember/optional-features": "^2.0.0",
+  "dependencies": {
     "@ember/test-helpers": "^2.6.0",
     "@ember/test-waiters": "^3.0.1",
+    "ember-modifier": "^3.2.0",
+    "ember-sortable": "workspace:../addon"
+  },
+  "dependenciesMeta": {
```

Comment:
because we need ember-sortable to behave _like a real dependency_ we need this injected setting. This allows the local workspace of ember-sortable to be treated as if it were actually from npm -- there is an open issue on the pnpm repo about making this _default_, because the behavior is most correct